spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
Just standardise raw. Once that's done this won't be necessary. Parent nodes should not be concerned about what their children represent (and vice versa), it's a part of the ESTree philisophy.

On Thu, Aug 13, 2015 at 10:07 PM, Ingvar Stepanyan
<notifications@github.com> wrote:

>> @@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
>>  
>>  An expression statement, i.e., a statement consisting of a single expression.
>>  
>> +## Directive
>> +
>> +```js
>> +interface Directive <: ExpressionStatement {
>> +    directive: string;
>> +}
>> +```
>> +
>> +A directive from the directive prologue of a script or function.
>> +The `directive` property is the raw string source of the directive.
>> +This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
> For non-parsers none of the cases simplifies generation, as both add complexity to AST (but, well, it's evolution). As for semantics - we are not trying to reflect semantics in AST at all, we are reflecting syntax (it's Syntax Tree after all) and thus in many cases map semantically different pieces of programs to identical ASTs.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/estree/estree/pull/99/files#r37025558